src/components/Link/Link.scss

```diff
@@ -24,6 +20,7 @@
   &:focus {
     $chrome-default-outline: rgba(0, 103, 244, 0.247) auto rem(4.5px);
     outline: var(--p-override-none, $chrome-default-outline);
+    text-decoration: underline;
```

Comment:
Added underline on focus and active for extra visibility that something is focused/active.